### PR TITLE
Add Swagger UI

### PR DESCRIPTION
**Pull request by nadyakuzmina**
- Added Swagger UI to run on admin path. 
- Implementation is a bit tricky, but all the steps are documented in the Java doc for `HelloWorldApplication.registerSwaggerAtAdminPath` method. 
- Insights into the implementation are at the following links:
https://www.reonomy.com/blog/post/augmenting-dropwizard-with-swagger
https://spin.atomicobject.com/2014/10/11/serving-static-assets-with-dropwizard/
- Adding swagger libraries entails a lot of exclusions, otherwise there are compilation errors of the 'ClassNotFound' type.
- Open Question:  Current Swagger configuration will make ALL endpoints (Resource endpoints, admin endpoints, internal(?) endpoints) available in Swagger. Execution of the admin endpoints does not work due to path issues. This is what our Swagger page looks like: 
swagger
Question: Do we want to try to limit Swagger to just our resource endpoints? I think it is probably possible via some config, but it will take some figuring out. The drawback is that we may need to be extending the config to include new resources in Swagger as we add them. What do you think, Greg?
My inclination is to leave Swagger as it is for now and worry about enhancing its config later if we would so wish.

### CODE REVIEW

File: HelloWorldService/src/main/java/com/gpnk/helloworld/HelloWorldResource.java [hunk 1]
```diff
@@ -55,7 +57,9 @@ public HelloWorldResource() {
      */
     @GET
     @Timed
-    public String sayHello(@QueryParam("name") final Optional<String> name) {
+    @Operation(description = "Says hello with the name, if the name is given. Outputs weather forecast for registered users.")
+    public String sayHello(@Parameter(description = "Name of the user", required = false)
+                           @QueryParam("name") final Optional<String> name) {
```
Comment (nadyakuzmina) [inline HelloWorldService/src/main/java/com/gpnk/helloworld/HelloWorldResource.java:62]: Looking for suggestions on other Swagger doc to include.

Comment (mrgreg) [reply HelloWorldService/src/main/java/com/gpnk/helloworld/HelloWorldResource.java:62]: I think this is fine for now.

**Review by mrgreg — APPROVED** [review]
This is good for now.

It's too bad we have to bring in all those static resources directly.  I wonder if, in the long run, we might be better off having each service generate and publish an openapi 3 spec.  And then have a centralized (internal to our company) way of viewing/calling them.  Perhaps a simple as a stand-alone swaggerUI instance, or maybe there's something better.  But I suggest we find a more robust solution when we need it.

**Comment by nadyakuzmina** [comment]
Yeah, copying Swagger UI dist into our code base isn't pretty. But as far as I can tell, this is what most solutions (leakycauldron, and https://www.reonomy.com/blog/post/augmenting-dropwizard-with-swagger) do. 
I agree, the approach you suggest sounds better. I will add a TODO to basecamp for it. 
For now, we go with the current implementation.

**Pull request merged**